### PR TITLE
tests: unittests: add tests for longer input vectors

### LINKED ISSUE #5299: tests: Fletcher test vectors are too short
https://github.com/RIOT-OS/RIOT/pull/4938#issuecomment-198878721

It might be a good idea for the fletcher tests to have a test input vector which accumulates enough to "wrap around" the modulo part of the algorithm. The current test vector is quite short and containing small numbers.

### PR DESCRIPTION
**Pull request by miri64**
Fixes #5299

### CODE REVIEW

File: tests/unittests/tests-checksum/tests-checksum-fletcher16.c [hunk 1]
```diff
@@ -14,6 +14,10 @@
 
 #include "tests-checksum.h"
 
+/* String is longer than 20 so it checks the wrap-around property of
+ * fletcher16 */
+static const unsigned char wrap_around_data[] = "u9k89rm88z58f3hGxTfgG3BnyKnyRP";
```
Comment (Kijewski) [inline tests/unittests/tests-checksum/tests-checksum-fletcher16.c]: Well, this won't work with properly on non-ASCII platforms, but I guess RIOT won't be ported to 30 year old IBM mainframes anytime soon, so this non-portability probably OK. :)

Comment (miri64) [inline tests/unittests/tests-checksum/tests-checksum-fletcher16.c]: If that is going to happen, we need to adapt the tests below anyways :black_joker:

Comment (miri64) [inline tests/unittests/tests-checksum/tests-checksum-fletcher16.c]: Would be probably best then to change this from `unsigned char` to `uint8_t`, even in the API ;-)

**Comment by x3ro** [comment]
ACK

**Pull request merged**